edgedb/legacy_transaction.py

```diff
@@ -0,0 +1,324 @@
+#
```

Comment:
Please add a comment clarifying why we have it -- essentially it's only needed to the legacy `con.transaction()` that's scheduled for removal soon.